### ISSUE TITLE
Как пофиксить трабл с запуском npm install -g generator -bem-stub@0.4.0?

### ISSUE THREAD

**Issue by mathetes** [bem/bem-forum-content-ru#432]
Практикуюсь по мастер классу Мастер-класс+наоборот+вы+пишете+БЭМ-проект,+а+мы+подсказываем+—+Евгений+Константинов,+Дима+Белицкий
Список 16 задач получил
Теперь надо запустить npm install -g generator -bem-stub@0.4.0. В итоге ругается на spawn-sync package
Смотрим скриншот https://yadi.sk/i/GIFiNVGtgyRDi

**Comment by kompolom** [comment]
Что то здесь не сходится. на скрине другая комманда

**Comment by mathetes** [comment]
Извиняюсь. Вот этот скрин https://yadi.sk/i/hWiDz2tsgyh9L

**Comment by kompolom** [comment]
В ubuntu глобальная установка пакетов, через sudo

**Comment by mathetes** [comment]
Делаю через sudo но увы...
Получаю Err
Прилагаю скриншот от начала команды с установкой генератора до результата
https://yadi.sk/i/txEiS7d9h3Bpr

**Comment by tadatuta** [comment]
@eGavr Жень, посмотришь?

**Comment by zxqfox** [comment]
Эти peer депсы... @mathetes удаляй все, на что он ругается, и ставь заново. Больше никак.

**Comment by eGavr** [comment]
Попробуй вариант @zxqfox, должно пофикситься

**Comment by eGavr** [comment]
@tadatuta @mathetes

С другой стороны, если это не поможет, то может стоит поставить последнюю версию генератора? там будут некоторые изменения в формулировке вопросов, но разобраться должно быть не трудно и на прохождение мастер-класса не должно влиять...

**Comment by tadatuta** [comment]
@mathetes удалось ли решить проблему?

**Comment by mathetes** [comment]
Да. Проблемы с установкой решились. Сейчас ставлю любой bem пакет без проблем. Как я понял проблема решилась через глобальную установку зависимостей и необходимых библиотек.
